src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -175,99 +206,99 @@ def export_weights(self):
-                x_data = _np.asarray(x_data).reshape((1, 28, 28, 1))
-                y_data = _np.asarray(y_data).reshape((1, self.num_classes))
-                acc = self.sess.run(self.accuracy,
-                            feed_dict={
-                                self.x: x_data,
-                                self.y: y_data
-                            })
-                total_acc+= acc
-                count+=1
-
-        final_accuracy = total_acc/count
-        return final_accuracy
-
-
-def _tf_train_model(tf_model, train_loader, validation_loader, validation_set, batch_size, num_classes, verbose):
-    """
-    Trains the TensorFlow model.
-
-    Returns
-    -------
-
-    final_train_accuracy : numpy.float32
-        Training accuracy of the last iteration.
-
-    final_val_accuracy : numpy.float32
-        Validation accuracy of the last iteration.
-
-    final_train_loss : numpy.float32
-        The final loss recorded in training.
-
-    total_train_time : float
-        Time taken to complete the training
-    """
-
-    if verbose:
-        column_names = ['iteration', 'train_loss', 'train_accuracy', 'time']
-        column_titles = ['Iteration', 'Training Loss', 'Training Accuracy', 'Elapsed Time (seconds)']
-        if validation_set is not None:
-            column_names.insert(3, 'validation_accuracy')
-            column_titles.insert(3, 'Validation Accuracy')
-        table_printer = _ProgressTablePrinter(column_names, column_titles)
-
-    num_iter = 0
-    start_time = _time.time()
-    for train_batch in train_loader:
-        batch_x, batch_y = process_data(train_batch, batch_size, num_classes)
-        result = tf_model.train( feed_dict={
-                            'x': batch_x,
-                            'y': batch_y
-                        })
-        final_train_accuracy = result['accuracy']
-        final_train_loss = result['loss']
-
-        for val_batch in validation_loader:
-            val_x, val_y = process_data(val_batch)
-            result = tf_model.predict(feed_dict={
-                            'x': val_x,
-                            'y': val_y
-                        })
-            val_acc = result['accuracy']
-        validation_loader.reset()
-
-        num_iter+=1
-        if verbose:
-            kwargs = {  "iteration": num_iter,
-                        "train_loss": "{:.3f}".format(final_train_loss),
-                        "train_accuracy": "{:.3f}".format(final_train_accuracy),
-                        "time": _time.time() - start_time}
-            if validation_set is not None:
-                kwargs["validation_accuracy"] = "{:.3f}".format(val_acc)
-            table_printer.print_row(**kwargs)
-
-    final_val_accuracy = val_acc if validation_set else None
-    total_train_time = _time.time() - start_time
-
-    return final_train_accuracy, final_val_accuracy, final_train_loss, total_train_time
+    # def evaluate(self, train_loader):
```

Comment:
I suggest we remove this portion completely as this was needed when tensorflow worked with Python implementation.